docs/learn/smart-contracts/smart-contract-execution.md

```diff
@@ -0,0 +1,79 @@
+---
+description: An in-depth exploration of smart contract execution in blockchain technology, detailing their structure, invocation, and associated fees.
+image: /img/logo/WASP_logo_dark.png
+keywords:
+- Smart Contracts
+- Blockchain Technology
+- Validators
+- User Requests
+- Gas Fees
+---
+
+# Smart Contract Execution
+
+## TL;DR;
+
+- [Introduction](#introduction): Smart contracts are self-executing contracts on blockchain networks, activated by validators in response to user requests.
```

Comment:
I'm not sure I understand what is meant with "self-executing contracts". I'd prefer to just call them programs, ran by validators upon request.

Reply:
```suggestion
- [Introduction](#introduction): Smart contracts are programs run by validators upon request by validators in response to user requests.
```

Reply:
Why is this resolved?

Reply:
don't know, it seems the suggestion didn't go through but it still resolved...weird. Anyhow, updated